web/src/pages/SearchNearbyPage/SearchNearbyPage.tsx

```diff
@@ -1,14 +1,26 @@
-import { Link, routes } from '@redwoodjs/router'
+import { useState } from 'react'
+
 import { MetaTags } from '@redwoodjs/web'
 
 import SearchNearby from 'src/components/discover/SearchNearby'
+import NearbyLocationsCell from 'src/components/NearbyLocationsCell'
 
 const SearchNearbyPage = () => {
+  const [location, setLocation] = useState<string>(null)
+  const radius = 8000
+
+  const handleLocation = (loc: { latitude: number; longitude: number }) => {
+    const coords = `${loc.latitude},${loc.longitude}`
+    setLocation(coords)
+    setLocation(coords)
+  }
+
   return (
     <>
       <MetaTags title="SearchNearby" description="SearchNearby page" />
 
-      <SearchNearby />
+      <SearchNearby onLocation={handleLocation} />
+      {location && <NearbyLocationsCell location={location} radius={radius} />}
```

Comment:
move this and `handleLocation` into `SearchNearby`. page logic should be minimal, mostly the metatags and wrapper around the various components - no real business logic